sdk/go/auto/local_workspace.go

```diff
@@ -73,7 +73,7 @@ func (l *LocalWorkspace) SaveProjectSettings(ctx context.Context, settings *work
 func (l *LocalWorkspace) StackSettings(ctx context.Context, stackName string) (*workspace.ProjectStack, error) {
 	name := getStackSettingsName(stackName)
 	for _, ext := range settingsExtensions {
-		stackPath := filepath.Join(l.WorkDir(), fmt.Sprintf("pulumi.%s%s", name, ext))
+		stackPath := filepath.Join(l.WorkDir(), fmt.Sprintf("Pulumi.%s%s", name, ext))
```

Comment:
Is this something that we need to address across languages?

Reply:
Sorry I didn't check the other SDKs. But if we are assuming case insensitivity then definitely.